templates/next-starter-with-examples/scripts/stackblitz-yarn.lock

```diff
@@ -0,0 +1,3528 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
when is this file generated?

Reply:
Never, I just did that in the different directory and then copied the file contents.

Reply:
But now I see the trouble it can cause. I will remove that file since updating it automatically on each version bump is not feasible. This was here only to speed up the installation.